backend/infrahub/menu/menu.py

```diff
@@ -135,6 +135,16 @@ def _extract_node_icon(model: MainSchemaTypes) -> str:
                 section=MenuSection.INTERNAL,
                 order_weight=1500,
             ),
+            MenuItemDefinition(
+                namespace="Builtin",
+                name="ObjectTemplates",
+                label="Object Templates",
```

Comment:
@lykinsbd would it make sense to just call it `Templates` instead of `Object Templates`

Reply:
This makes sense to me as we don't have any other kind of templates and we don't call it "object profiles".